sample-config-files/aad-config.yaml

```diff
@@ -17,7 +17,7 @@ Description: |
   - Namespace for each policy item within a product for variables related only to one policy item (i.e., Aad.Policy2_1)
   - Use of YAML anchors and aliases following DRY (Don't Repeat Yourself) principle for repeated values and sections
 ProductNames:
-  - aad
+- aad
```

Comment:
I don't see any changes to this file that affects the code. What is the purpose of these changes?